queries.sql

```diff
@@ -7,4 +7,51 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Tuesday */
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+--1st transaction
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon'
+WHERE species IS NULL;
+--2nd transaction
+COMMIT;
+SELECT * FROM animals ORDER BY id;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+--3rd transaction
+ROLLBACK;
+SELECT * FROM animals ORDER BY id;
+
+BEGIN;
+DELETE FROM animals
+WHERE date_of_birth > DATE '2022-01-01';
+SAVEPOINT animals_after_JAN2022;
+SELECT * FROM animals ORDER BY id;
+UPDATE animals SET weight_kg = weight_kg * -1;
+SELECT * FROM animals ORDER BY id;
+ROLLBACK TO animals_after_JAN2022;
+UPDATE animals SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+--4th transaction
+COMMIT;
+SELECT * FROM animals ORDER BY id;
+
+/* Aggregates */
+SELECT * from animals;
+SELECT COUNT(*) AS "Animals_Count" FROM animals;
+SELECT COUNT(*) AS "Escapee_Animals_Count" FROM animals WHERE escape_attempts = 0;
```

Comment:
- [ ] The result returned for this query is `2` and not `1`. `Charmander` and `Agumon` never tried to escape. The statement is correct but the data inputted has errors. Kindly input the correct data and update your screenshot. 👍